jekyll/_cci2/build-processing.md

```diff
@@ -17,26 +17,23 @@ This document describes how to enable the pipelines engine if you need to trigge
 
 Most projects will have Pipelines enabled by default. Verify the project pipeline setting in the Advanced section of your project's Settings page in the CircleCI app. **Note:** Pipelines are compatible with v2 and v2.1 configurations of CircleCI.
 
-## Benefits of Pipelines
+## What are Pipelines?
 
-The pipelines feature enables use of the new [API endpoint to trigger builds with workflows](https://circleci.com/docs/api/#trigger-a-new-build-by-project-preview) and the following use cases:
+Pipelines are our name for the full set of configurations you run when you trigger work on your projects in CircleCI. Pipelines contain your workflows, which coordinate jobs.
 
-{% include snippets/pipelines-benefits.adoc %}
+## Benefits of Using Pipelines
 
-## Troubleshooting
+The pipelines feature enables use of the new [API endpoint to trigger builds with workflows](https://circleci.com/docs/api/#trigger-a-new-build-by-project-preview) and the following use cases:
 
-Pipeline errors will appear on the Jobs page or the Workflows page.
+{% include snippets/pipelines-benefits.adoc %}
 
-Please note that once pipelines are enabled for a project on a usage plan, they may not be
-turned off without filing a ticket with CircleCI support. See `limitations` below.
+## Implications of Using Pipelines
 
-## Limitations
-CircleCI is committed to achieving backwards compatibility in almost all cases, and most projects that enable pipelines will have no effect on existing builds. Let us know if you experience breaking builds that worked before you turned on pipelines, but broke after you turned it on.
+When using pipelines, please note the following implications of pipelines:
 
-- Anchors will be processed and resolved instead of appearing in the app config.
-- If you use `<<` in your shell commands (most commonly found in use of heredocs) you will need to escape them using backslash `\` as in `\<<` in order to use version 2.1 or higher of configuration. 2.0 configuration will not be affected.
-- Pipelines are **not** fully backwards-compatible with the 1.1 API endpoint to trigger arbitrary jobs - you may experience unexpected or inconsistent results if you use this endpoint after turning on Pipelines. Alternatively, you can use the [build-triggering endpoint in the 1.1 API](https://circleci.com/docs/api/#trigger-a-new-build-by-project-preview) introduced in September 2018. Please keep in mind that this build triggering API endpoint does **not** accept parameters or workflow or job filters. If you make heavy use of those API features and want to use Pipelines, please contact your CircleCI account team.
-- Configuration version 2.0 will have the `CIRCLE_COMPARE_URL` environment variable injected into all jobs for backwards compatibility.
+- Both 2.0 and 2.1 configurations wrap the 'build' job in a workflow if no 'workflows' are defined.
+- API 1.1 job triggering works via the Access Control List (ACL) for both 2.0 and 2.1 configurations.
```

Comment:
Not sure if we need to call this out, we can probably remove this one.

Reply:
Removed from document.